testproject/Assets/Tests/Runtime/MultiprocessRuntime/Helpers/BuildMultiprocessTestPlayer.cs

```diff
@@ -0,0 +1,117 @@
+using System;
+using System.IO;
+#if UNITY_EDITOR
+using UnityEditor;
+using UnityEditor.Build.Reporting;
+#endif
+using UnityEngine;
+
+/// <summary>
+/// This is needed as Unity throws "An abnormal situation has occurred: the PlayerLoop internal function has been called recursively. Please contact Customer Support with a sample project so that we can reproduce the problem and troubleshoot it."
+/// when trying to build from Setup() steps in tests.
+/// </summary>
+public class BuildMultiprocessTestPlayer : MonoBehaviour
+{
+    public const string multiprocessBaseMenuName = "MLAPI Multiprocess Test";
+    public const string BuildAndExecuteMenuName = multiprocessBaseMenuName + "/Build - Execute multiprocess tests #%t";
+    public static string buildPath => Path.Combine(Path.GetDirectoryName(Application.dataPath), "Builds/MultiprocessTestBuild");
+    public const string mainSceneName = "MultiprocessTestingScene";
+
+
+#if UNITY_EDITOR
+    [MenuItem(multiprocessBaseMenuName+"/Build Test Player #t")]
+    public static void BuildNoDebug()
+    {
+        var success = Build();
+        if (!success)
+        {
+            throw new Exception("Build failed!");
```

Comment:
The only plans for usage is either through yamato scripts or through manual menu clicking.
Do you see this being used by other C# code where it could have the possibility to be caught there?

Reply:
```suggestion
            throw new Exception($"Build did not succeed! ({buildResult})");
```

Reply:
```suggestion
            throw new Exception($"Build did not succeed! ({buildResult})");
```